packages/tupaia-web/src/features/Map/MapOverlays/DataVisualsLayer.tsx

```diff
@@ -73,11 +74,14 @@ const useEntitiesByMeasureLevel = (measureLevel?: string) => {
   );
 };
 
-export const DataVisualsLayer = ({
-  hiddenValues,
-}: {
-  hiddenValues: LegendProps['hiddenValues'];
-}) => {
+export const DataVisualsLayer = (
+  { isOpen, setIsOpen }: any,
```

Comment:
I don't think we need `DataVisualsLayer` to know anything about the `MapTableModal`, I think all the logic in it should be contained within `MapTableModal` (see other comments for more on this)